js/adapt-contrib-xapi.js

```diff
@@ -1349,7 +1349,18 @@ define([
         return;
       }
 
-      Adapt.trigger('notify:alert', { title: this.getGlobals().lrsConnectionErrorTitle, body: this.getGlobals().lrsConnectionErrorMessage, confirmText: this.getGlobals().confirm });
+      var notifyObject = {
+        title: this.getGlobals().lrsConnectionErrorTitle,
+        body: this.getGlobals().lrsConnectionErrorMessage,
+        confirmText: this.getGlobals().confirm
+      };
+
+      // Setup wait so that notify does not get dismissed when the page loads
+      Adapt.wait.begin();
+      Adapt.trigger('notify:alert', notifyObject);
+      // Ensure notify appears on top of the loading screen
+      $(".notify").css({position: "relative", zIndex: 5001});
```

Comment:
Minor: 
- spacing after/before the braces
- using single quotes for the "relative" string